Qa/kommander 1 1 azure create (#3022)
mesosphere/dcos-docs-site: pages/ksphere/kommander/1.1.0-beta/clusters/creating-konvoy-cluster-azure/index.md

- **Kubernetes Version**: The initial version of Kubernetes to install on the cluster.
- **Name**: A display name for referencing the cluster.

> Now select the preconfigured [Azure Infrastructure Provider](/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/configure-azure-infrastructure-provider/) to display the remaining options specific to Azure.

Reviewer: ```suggestion
Select the preconfigured [Azure Infrastructure Provider](/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/configure-azure-infrastructure-provider/) to display the remaining options specific to Azure.
```

> - **Region**: Select the datacenter region to reveal selectable Availability Zones. <a href="https://azure.microsoft.com/en-us/global-infrastructure/regions/" target="_blank">Azure maintains multiple datacenters per region</a>.

Reviewer: ```suggestion
- **Region**: Select the datacenter region to show selectable Availability Zones. <a href="https://azure.microsoft.com/en-us/global-infrastructure/regions/" target="_blank">Azure maintains multiple datacenters per region</a>.
```

- **Resource Group**: The resource group where the Azure assets are assigned.
- **VNET Name**: The virtual network name in which to create this cluster.

> - **VNET Route Table**: Azure automatically routes traffic between Azure subnets, virtual networks, and on-premises networks. If you want to change any of Azure's default routing, you do so by creating a <a href="https://docs.microsoft.com/en-us/azure/virtual-network/manage-route-table" target="_blank">route table</a>.

Reviewer: ```suggestion
- **VNET Route Table**: Azure automatically routes traffic between Azure subnets, virtual networks, and on-premises networks. To change any of Azure's default routing, create a <a href="https://docs.microsoft.com/en-us/azure/virtual-network/manage-route-table" target="_blank">route table</a>.
```

- **Configure Node Pools**: Specify pools of nodes, their size, and quantity. The suggested default is a control plane pool containing 3 Standard_DS3_v2 nodes and a worker pool containing 6 Standard_DS3_v2 nodes. You can also specify labels, node taints, and subnet ID's as part of each pool.

> - **Add Infrastructure Provider Tags**: Specify tags on all resources created in your infrastructure provider for this cluster. You can add up to 10 tags in this form. Different infrastructure providers have varying restrictions on their tags. See the <a href="https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources" target="_blank">Azure Tags User Guide</a> for more information about using tags in Azure.

Reviewer: ```suggestion
- **Add Infrastructure Provider Tags**: Specify tags on all resources created, in your infrastructure provider, for this cluster. You can add up to 10 tags in this form. Different infrastructure providers have varying restrictions on their tags. See the <a href="https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources" target="_blank">Azure Tags User Guide</a> for more information on using tags in Azure.
```
